pkg/volume/flexvolume/driver-call.go

```diff
@@ -235,3 +241,29 @@ func handleCmdResponse(cmd string, output []byte) (*DriverStatus, error) {
 
 	return &status, nil
 }
+
+// getDriverCapabilities returns the reported capabilities as returned by driver's init() function
+func getDriverCapabilities(plugin *flexVolumePlugin) (*driverCapabilities, error) {
+	call := plugin.NewDriverCall(initCmd)
```

Comment:
Not a merge blocker, but @chakri-nelluri could there be an issue calling driver init twice (since it's also called in Init())?